TFLite/src/TFLiteParser.py

```diff
@@ -0,0 +1,133 @@
+import flatbuffers
```

Comment:
This looks great, I think finalizing the schema, and commiting to Spanner, running this to generate features for the hosted models on tflite and populating and write a few useful queries would be great to have as a e2e milestone for mid-point.